docs/docs/addons-perf.md

```diff
@@ -57,6 +62,34 @@ The following methods use the measurements returned by [`Perf.getLastMeasurement
 
 * * *
 
+## Example Usage
+
+We will take simple example of a form which can be used to submit comments and then listing all comments. The example code is [available on GitHub](https://github.com/dhyey35/react-example-for-performance/blob/master/public/scripts/example.js).
+
+```javascript
+import { Component } from 'react'
+import Perf from 'react-addons-perf'
+
+const Comment = (props) => (
+	<div className="comment">
+	  <h2 className="commentAuthor">
+	    {props.author}
+	  </h2>
+	  <span>{props.children.toString()}</span>
+	</div>
+ )
+//code...
```

Comment:
What does this comment represent? I don't think it makes the example more clear.